ndb/memcache_client.py

```diff
@@ -0,0 +1,220 @@
+#
+# Copyright 2017 The ndb Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from .google_imports import namespace_manager
+from .google_imports import memcache
+
+from . import autobatcher
+from . import tasklets
+
+class MemcacheClient(object):
+  # NOTE: The default memcache prefix is altered if an incompatible change is
+  # required. Remember to check release notes when using a custom prefix.
+  _memcache_prefix = 'NDB9:'  # TODO: Might make this configurable
```

Comment:
It seems odd that the prefix is defined, but afaict, not actually used in this file.

Reply:
Good point. It's only used in `Context`. Remvoed.

Reply:
I don't think this PR has picked up that change.

Reply:
Sorry about that. Wrong branch. >_>